ui/app/templates/components/job-page/parts/summary.hbs

```diff
@@ -33,17 +40,25 @@
     </div>
   </a.head>
   <a.body>
-    {{#component (if a.item.hasChildren "children-status-bar" "allocation-status-bar")
-    allocationContainer=a.item.summary
-    job=a.item.summary
-    class="split-view" as |chart|}}
-    <ol data-test-legend class="legend">
-      {{#each chart.data as |datum index|}}
-        <li class="{{datum.className}} {{if (eq datum.label chart.activeDatum.label) "is-active"}} {{if (eq datum.value 0) "is-empty"}}">
-          <JobPage::Parts::SummaryLegendItem @datum={{datum}} @index={{index}} />
-        </li>
-      {{/each}}
-    </ol>
-  {{/component}}
+    {{#component
+      (if a.item.hasChildren "children-status-bar" "allocation-status-bar")
+      allocationContainer=a.item.summary
+      job=a.item.summary
+      class="split-view" as |chart|
+    }}
+      <ol data-test-legend class="legend">
+        {{#each chart.data as |datum index|}}
+          <li
+            class="{{datum.className}}
+
+              {{if (eq datum.label chart.activeDatum.label) "is-active"}}
+
+              {{if (eq datum.value 0) "is-empty"}}"
+          >
+            <JobPage::Parts::SummaryLegendItem @datum={{datum}} @index={{index}} />
+          </li>
```

Comment:
Was this auto-formatted? Looks a bit weird with the line breaks 🤔

Reply:
Actually, was anything even changed in this file apart from formatting? I can't spot any difference.